flow/frame_timings.h

```diff
@@ -121,6 +122,8 @@ class FrameTimingsRecorder {
   ///
   /// Instead of adding a `GetState` method and asserting on the result, this
   /// method prevents other logic from relying on the state.
+  ///
+  /// In opt builds, this call is a no-op.
```

Comment:
I believe we support optimized debug builds. Consider:

```suggestion
  /// In release builds, this call is a no-op.
```